tests/models/test_dagrun.py

```diff
@@ -142,6 +142,29 @@ def test_dagrun_find(self):
         assert 0 == len(models.DagRun.find(dag_id=dag_id2, external_trigger=True))
         assert 1 == len(models.DagRun.find(dag_id=dag_id2, external_trigger=False))
 
+    def test_dagrun_find_duplicate(self):
+        session = settings.Session()
+        now = timezone.utcnow()
+
+        dag_id1 = "test_dagrun_find_duplicate"
```

Comment:
```suggestion
        dag_id = "test_dagrun_find_duplicate"
```

nit: just a better variable name